urllib3/contrib/appengine.py

```diff
@@ -189,6 +189,10 @@ def _get_absolute_timeout(self, timeout):
                 warnings.warn(
                     "URLFetch does not support granular timeout settings, "
                     "reverting to total timeout.", AppEnginePlatformWarning)
+            if not timeout.total:
```

Comment:
Should this be `if timeout.total is None`? It's not totally beyond possibility that someone will set the timeout here to `0`, or `0.0`, or something of that kind.